Source/WebCore/rendering/RenderTable.cpp

```diff
@@ -1593,7 +1593,11 @@ std::optional<LayoutUnit> RenderTable::firstLineBaseline() const
     if (auto baseline = topNonEmptySection->firstLineBaseline())
         return std::optional<LayoutUnit>(topNonEmptySection->logicalTop() + baseline.value());
 
-    // FIXME: A table row always has a baseline per CSS 2.1. Will this return the right value?
+    // FF, Presto and IE use the top of the section as the baseline if its first row is empty of cells or content.
```

Comment:
both Presto (Opera) and Trident (IE) have disappeared. Maybe the reference to these is not useful. We could simply say "Other browsers".

Reply:
Yep - I kept it to make it easy in future to find any relevant Blink merges but will update once it get r+ before merging.

Reply:
Done. :-)